From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Sat, 4 Jun 2011 17:30:33 -0400
Subject: [PATCH] fix base path

---
 configure/RELEASE |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index 58d1e67..b7bbf52 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -21,5 +21,5 @@ TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 ## Required Modules ##
 
 # EPICS_BASE usually appears last so other apps can override stuff:
-EPICS_BASE=/usr/epics/base
+EPICS_BASE=/usr/lib/epics
 
-- 
