From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 11 May 2011 15:03:24 -0400
Subject: [PATCH] fix warning on linux 64

---
 vmeApp/vmesh.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vmeApp/vmesh.c b/vmeApp/vmesh.c
index a41cc37..4836c9f 100644
--- a/vmeApp/vmesh.c
+++ b/vmeApp/vmesh.c
@@ -71,7 +71,7 @@ void vmeread(epicsUInt32 addr, int amod, int dmod, int count)
     return;
   }
 
-  epicsPrintf("Mapped to 0x%08x for %d bytes\n",(unsigned int)mptr,dbytes*count);
+  epicsPrintf("Mapped to 0x%08lx for %d bytes\n",(unsigned long)mptr,dbytes*count);
 
   if( devReadProbe(
     dbytes,
-- 
